--- orkg_client/models/comparison_table_row_representation.py
+++ orkg_client/models/comparison_table_row_representation.py
@@ -105,10 +105,9 @@
             }
             if obj.get("children") is not None
             else None,
-            "values": [ThingReferenceRepresentation.from_dict(_item) for _item in obj["values"]] if obj.get("values") is not None else None
+            "values": [ThingReferenceRepresentation.from_dict(_item) if _item is not None else None for _item in obj["values"]] if obj.get("values") is not None else None
         })
         return _obj
 
 # TODO: Rewrite to not use raise_errors
 ComparisonTableRowRepresentation.model_rebuild(raise_errors=False)
-